### PR TITLE
Add Socks to Network Protocol Section

### PR DESCRIPTION
**Pull request by wontruefree**
This is the beginning of a Socks implementation I wanted to add to the Network Protocol section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -382,6 +382,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [OAuth](https://crystal-lang.org/api/OAuth.html) - OAuth consumer (Crystal stdlib)
  * [OAuth2](https://crystal-lang.org/api/OAuth2.html) - OAuth2 client (Crystal stdlib)
  * [OpenSSL](https://crystal-lang.org/api/OpenSSL.html) - bindings to libssl (Crystal stdlib)
+ * [Socks](www.sockscr.com) - SOCKS proxy
```
Comment (veelenga) [inline README.md]: Can we use gitlab link instead? Github/Gitlab ones are validated for uniqueness by tests.

Comment (wontruefree) [reply README.md]: I can. I was going to make a homepage with docs and stuff.

**Comment by wontruefree** [comment]
@veelenga is there something more I can do to get this merged?

**Comment by veelenga** [comment]
@wontruefree no, we are waiting for a second approval @hugoabonizio 

Sorry, for delay.

**Comment by hugoabonizio** [comment]
I missed that, sorry!

**Comment by wontruefree** [comment]
thanks :)

**Pull request merged**